### PR TITLE
Remove short since it's duplicate...

### PR DESCRIPTION
**Pull request by ROunofF**
## Pull Request Checklist

Reference : #50

#### General

- [X] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [X] Update README with any necessary configuration snippets

- [x] Binstubs are created if needed

- [X] RuboCop passes

- [X] Existing tests pass 

#### New Plugins

- [x] Tests

- [x] Add the plugin to the README

- [x] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachangelog.com/)
 
 ## [Unreleased]
+- Remove short options -s on hostname_sub, it was already taken
```
Comment (majormoses) [inline CHANGELOG.md]: Lets put this under a section with `### Breaking Changes` to be nice and loud about this. Also lets indicate which checks this affects.

**Review by majormoses — CHANGES_REQUESTED** [review]
Overall looks good, just a bit more clear changelog messages should be given when breaking changes are done.

File: CHANGELOG.md (hunk 1 above)
Comment (ROunofF) [reply CHANGELOG.md]: Hmmm the -s for host_sub was simply not usable... Unless I'm missing something?

Output when adding a puts config in options.rb:
`$ check-graphite-data.rb -s localhost -t test

{:name=>"graphite check", :from=>"-10mins", :until=>"now", :allowed_graphite_age=>60, :server=>"localhost", :target=>"test"}
CheckGraphiteData OK: graphite check value OK
`

Comment (majormoses) [reply CHANGELOG.md]: fair enough, put it under a `### Fixed` then.

Comment (majormoses) [reply CHANGELOG.md]: also include the affected checks.

**Review by majormoses — APPROVED** [review]

**Comment by majormoses** [comment]
released: https://rubygems.org/gems/sensu-plugins-graphite/versions/2.2.1

**Pull request merged**